webpack/assets/javascripts/react_app/components/FactCharts/__test__/__snapshots__/FactChartReducer.test.js.snap

```diff
@@ -0,0 +1,66 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`FactChart reducer initial state 1`] = `
+Object {
+  "chartData": Array [],
+  "loaderStatus": "",
+  "modalToDisplay": Object {},
+}
+`;
+
+exports[`FactChart reducer should handle FACT_CHART_DATA_ERROR action 1`] = `
+Object {
```

Comment:
should there be something specific to an error payload ?

Reply:
Yes, fixed the test, there is `  "loaderStatus": "ERROR",` now. Thanks!